### ISSUE TITLE
Documentation request

### ISSUE THREAD

**Issue by KameniAlexNea** [kscanne/chichewa#1]
Hello and thank you for this repository.
Can I have the documentation for the various files on your repo, please?

**Comment by neel04** [comment]
@kscanne agreed, documentation would be really useful to understand how we can use this repo

**Comment by kscanne** [comment]
Hello to you both.  I just added a basic README explaining a bit about what's here. I'm not sure there's much in the repo that will be useful to build on, but if you let me know what you're working on, maybe I can offer advice or pointers.
